tools/src/main/java/io/github/ygojson/tools/dataprovider/impl/yugipedia/model/CardTable2.java

```diff
@@ -0,0 +1,129 @@
+package io.github.ygojson.tools.dataprovider.impl.yugipedia.model;
+
+/**
+ * Yugipedia's Model for CardTable2.
+ * <br>
+ * {@link MarkupString} is used where markup content might be present.
+ * The rest of fileds are typed as {@link String} to prevent issues
+ * while typing values with special markers (i.e., atk/def values as '?'
+ * cannot be typed as a numeric).
+ *
+ * @see <a href=https://yugipedia.com/wiki/Template:CardTable2>CardTable2</a>
+ */
+public record CardTable2(
+	String cardclass,
+	String name,
+	String fr_name,
+	String de_name,
+	String it_name,
+	String es_name,
+	MarkupString ja_name,
+	String romaji_name,
+	String trans_name,
+	String base_romaji_name,
+	String base_trans_name,
+	String zh_name,
+	String zh_pinyin_name,
+	String zh_trans_name,
+	String ko_name,
+	String ko_rr_name,
+	String ko_trans_name,
+	String alt_name,
+	String alt_name2,
+	String alt_name3,
+	String ja_alt_name,
+	String ja_alt_romaji,
+	String ja_alt_trans,
+	String image,
+	String image2,
+	String card_type,
+	String property,
+	String attribute,
+	String character,
+	String types,
+	String level,
+	String rank,
+	String link_arrows,
+	String pendulum_scale,
+	String maximum_atk,
+	String atk,
+	String def,
+	String password,
+	String limitation_text,
+	String ritualcard,
+	String ritualmonster,
+	String summon,
+	String effect_types,
+	String pendulum_effect_types,
+	String ocg_status,
+	String tcg_status,
+	String tcg_speed_duel_status,
+	String source_card,
+	String skill_activation,
+	String fr_skill_activation,
+	String de_skill_activation,
+	String it_skill_activation,
+	String pt_skill_activation,
+	String es_skill_activation,
+	MarkupString pendulum_effect,
+	MarkupString fr_pendulum_effect,
+	MarkupString de_pendulum_effect,
+	MarkupString it_pendulum_effect,
+	MarkupString pt_pendulum_effect,
+	MarkupString es_pendulum_effect,
+	MarkupString ja_pendulum_effect,
+	MarkupString trans_pendulum_effect,
+	MarkupString zh_pendulum_effect,
+	MarkupString ko_pendulum_effect,
+	MarkupString lore,
+	MarkupString fr_lore,
+	MarkupString de_lore,
+	MarkupString it_lore,
+	MarkupString pt_lore,
+	MarkupString es_lore,
+	MarkupString ja_lore,
+	MarkupString trans_lore,
+	MarkupString zh_lore,
+	MarkupString ko_lore,
+	String use,
+	String database_id,
+	MarkupString en_sets,
+	MarkupString na_sets,
+	MarkupString eu_sets,
+	MarkupString au_sets,
+	MarkupString fr_sets,
+	MarkupString fc_sets,
+	MarkupString de_sets,
+	MarkupString it_sets,
+	MarkupString pt_sets,
+	MarkupString sp_sets,
+	MarkupString jp_sets,
+	MarkupString ja_sets,
+	MarkupString ae_sets,
+	MarkupString tc_sets,
+	MarkupString sc_sets,
+	MarkupString kr_sets,
+	MarkupString supports,
+	//  anti-supports_archetypes - as it is not a valid name
```

Comment:
Change these comments for a json-annotation, which is useful both as documentation and to output something more readable (and for a custom incoming tool for debugging purposes and to provide to the community).